placement/algo/sharded_helper.go

```diff
@@ -354,14 +389,7 @@ func (ph *placementHelper) PlaceShards(
 		// NB(cw) when removing an adding instance that has not finished bootstrapping its
 		// Initializing shards, prefer to return those Initializing shards back to the leaving instance
 		// to reduce some bootstrapping work in the cluster.
-		if err := ph.returnInitializingShardsToSource(shardSet, from, candidates); err != nil {
-			return err
-		}
-		// prefer to distribute "some" of the load to other racks first
```

Comment:
This is no longer needed?

Reply:
No longer needed, we use Optimize() to achieve better load distribution now.